docs/ci_cd_process.md

```diff
@@ -0,0 +1,28 @@
+# CI / CD process
+
+This document aims to explain everything related to Continuous Integration and Continuous Development (CI / CD) processes of the app.
+
+
+## Deployment process
+
+This app is in current development under the `develop` branch. The `master` branch is currently empty, because we have not yet put this into production (however this is in the works).
+
+Currently we have the development version oof this app deployed on [Heroku]. Because we have "Automatic Deploys" feature  enabled, everytime a new pull request is merged into `develop` branch a new version of the app is deployed to [Heroku].
```

Comment:
```suggestion
Currently, we have the development version of this app deployed on [Heroku]. Because we have "Automatic Deploys" feature enabled, every time a new pull request is merged into `develop` branch a new version of the app is deployed to [Heroku].
```